draft-ietf-tls-tls13.md

```diff
@@ -2957,8 +2956,17 @@ the responses).
 
 
 ### Key and IV Update {#key-update}
+     enum { update_not_requested(0), update_requested(1), (255)
```

Comment:
Oh I meant that more suggesting that we should find a way to denote it in the enum.